wgpu/Cargo.toml

```diff
@@ -42,6 +42,11 @@ optional = true
 package = "wgpu-types"
 path = "../wgpu-types"
 
+[target.'cfg(target_arch = "wasm32")'.dependencies.hal]
```

Comment:
this shouldn't be required. If there is anything wgpu needs from hal, it needs to be re-exported by wgpu-core